src/libopenrave-core/colladaparser/colladareader.cpp

```diff
@@ -1056,11 +1056,17 @@ class ColladaReader : public daeErrorHandler
             if( !!pinterface_type ) {
                 if( pinterface_type->type == "kinbody" ) {
                     pbody = RaveCreateKinBody(_penv,pinterface_type->name);
+                    if (!pbody) {
```

Comment:
These 2 new `if`s are requested by Osbert and Guangyang. They have members who cannot access the internal plugins, so cannot load vendor specific robot interface type. They want to in that case load the default robot class so they can use OpenRAVE to continue to edit scenes.

Reply:
@ziyan can you give a warning message then?

Reply:
Done.